tls/tls.md

```diff
@@ -116,8 +121,15 @@ message PublicKey {
 
 **TODO: PublicKey.Data looks underspecified. Define precisely how to marshal the key.**
 
-## Future Extensibility
+## SNI
+Clients MAY use the Server Name Indication (SNI) in the `ClientHello` as defined in [RFC 6066 §3](https://www.rfc-editor.org/rfc/rfc6066#section-3).
+
+
+# Changelog
+## V1.1.0
 
-Future versions of this handshake protocol MAY use the Server Name Indication (SNI) in the `ClientHello` as defined in [RFC 6066, section 3](https://tools.ietf.org/html/rfc6066) to announce their support for other versions.
+Allow Clients to send an [SNI](#sni). Servers MAY use the SNI.
+### Backwards compatibility
 
-In order to keep this flexibility for future versions, clients that only support the version of the handshake defined in this document MUST NOT send any value in the Server Name Indication. Servers that support only this version MUST ignore this field if present.
+- Clients communicating with nodes only speaking `v1.0.0` MUST NOT send an SNI.
+- Servers communicating with nodes only speaking `v1.0.0` MUST ignore the SNI.
```

Comment:
This is confusing me a bit. Why would a `v1.0.0` client send an SNI?

I don't think this directly translates from the previous version:

> In order to keep this flexibility for future versions, clients that only support the version of the handshake defined in this document MUST NOT send any value in the Server Name Indication. Servers that support only this version MUST ignore this field if present.

Wouldn't the following be more accurate?

```
- Clients supporting `v1.0.0` only MUST NOT send an SNI.
- Servers supporting `v1.0.0` MUST ignore the SNI.
- ```